src/customHook/CustomHook.jsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+
+function useCustomHooks() {
```

Comment:
this is wrong . the name of fucntion should not be custom hook. 

it should be as per the state. like useTodoForm something liek this.

Reply:
ok sir,